### ISSUE TITLE
Screen Reader issue in JQGrid version 4.4.0

### ISSUE THREAD

**Issue by Raunak0786** [tonytomov/jqGrid#955]
**Problem** :User who depends upon screen reader won’t be able to get table information properly if screen reader doesn’t announced the associate column header with cell value and also they won’t be understand the context of table. 

**Repro Steps**: 
1. Open any JQGrid version 4.4.0
2. Navigate to table
3. Verify below scenario 
   a)Tables not reading the column header association with along with the cell value.
   b) Incorrect Row count Information.
   c) Screen reader is also not passing the information for the sorting.

**Expected Result**:
   a) Screen reader properly read column header on every column cell
  b) Screen reader must reader correct row and column name with table command eg: 'T'
  c) Screen reader must pass the information for the sorting.

#JQGrid

**Comment by tonytomov** [comment]
The current supported version is 5.4.  The version 4.4 is seven(7) years old and is not supported. Sorry.

**Comment by Raunak0786** [comment]
Hi Tony,

We also tested the current JQGrid version 5.4.0 and found that below scenario is not fix in JQgrid

a)Tables not reading the column header association with along with the cell value.
b) Incorrect Row count Information.
c) Screen reader is also not passing the information for the sorting.

**Comment by tonytomov** [comment]
Since there are a lot of different behaviours, please specify the OS, the browser used and the name of the screen reader. Example With the last version of Narrator in Win10 and MS Edge nothing work - with the previous version everthing work as expected.

I recommend you to read [this discussion](http://guriddo.net/?topic=regarding-accessibility-feature-in-jqgrid)
